TeacherWorkout.Api/Startup.cs

```diff
@@ -36,6 +37,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllers();
             
             services.AddSingleton<TeacherWorkoutQuery>();
+            services.AddSingleton<TeacherWorkoutMutation>();
+            services.AddSingleton<LessonSaveInput>();
```

Comment:
Lets update the `AddGraphTypes` method to automatically register all input types.